easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'DFTB+'
+version = '1.3.1'
+
+homepage = 'https://www.dftb-plus.info'
+description = """DFTB+ is a fast and efficient versatile quantum mechanical simulation package.  It is based on the 
+ Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been
+ developed for the DFTB framework so far.  Using DFTB+ you can carry out quantum mechanical simulations like with
+ ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of
+ magnitude in speed."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://www.dftb-plus.info/download/dftb-13']  # requires registration
+sources = [
+    'dftbplus-%(version)s-src.tar.xz',
+    'dftbplus-%(version)s-extlib.tar.xz',
+    'dftbplus-%(version)s-autotest.tar.xz',
+]
+
+checksums = [
+    'b821ed7406a2a4b35c0a48974e18667f',  # dftbplus-1.3.1-src.tar.xz
+    '2333f1ff826a3c4776e1e77a28f17004',  # dftbplus-1.3.1-extlib.tar.xz
+    'abaf8a2f94c74a13a0d8d19e9b022e40',  # dftbplus-1.3.1-autotest.tar.xz
+]
+
+dependencies = [
+    ('arpack-ng', '3.4.0'),
+]
+
+subdir = '%dftbplus-%(version)s'
+
+prebuildopts = "sed 's/ARCH = x86_64-linux-intel/ARCH = user/' Makefile.user.template > Makefile.user && "
+prebuildopts += "cp sysmakes/make.x86_64-linux-intel sysmakes/make.user && "
+prebuildopts += "sed -i 's|LNOPT =  -static|#LNOPT =  -static|' sysmakes/make.user && "
+prebuildopts += "sed -i 's|/opt/intel/mkl/10.2.6.038/lib/em64t|$(MKLROOT)|' sysmakes/make.user && "
```

Comment:
@migueldiascosta are these `sed` commands really needed? Can't you just override these values with arguments to `make` via `buildopts`?

```python
buildopts = 'ARCH=user LNOPT= '
```